src/main/webapp/i18n/de/guidedTourCodeEditor.json

```diff
@@ -0,0 +1,47 @@
+{
+    "tour": {
+        "programmingExercise": {
+            "fileBrowser": {
+                "headline": "Datei Browser",
+                "content": "Im Datei Browser findest die Struktur des Projekts mit allen Ordnern und Dateien. Du kannst hier auch eigene Ordner und Dateien anlegen."
+            },
+            "exerciseFile": {
+                "headline": "Klassendatei",
+                "content": "Klick hier auf die Klassendatei, damit sie sich im Programmier-Editor öffnet."
+            },
+            "codeEditor": {
+                "headline": "Programmier-Editor",
+                "content": "Nun wird dir der Quellcode im Programmier-Editor angezeigt. Du kannst dir das TODO durchlesen und bereits versuchen, dieses zu lösen. Alternativ kannst du auch jeden beliebigen Code zum in dieser Testrunde einfügen. Wenn du damit fertig bist, dann klicke auf weiter."
+            },
+            "saveChanges": {
+                "headline": "Änderungen speichern",
+                "content": "Du kannst deine Änderungen im Programmier-Editor nun speichern, indem du auf Speichern klickst."
+            },
+            "submitChanges": {
+                "headline": "Änderungen absenden",
+                "content": "Wenn du mit deinem Code zufrieden bist, dann kannst du deinen ersten Stand einreichen, indem du auf 'Absenden' klickst.",
+                "hint": "Hier noch ein kleiner Hinweis: Du kannst bis zum Abgabedatum mehrfache Abgaben machen, das heißt du kannst deine Aufgaben einzeln lösen und danach einreichen."
+            },
+            "buildLoading": {
+                "headline": "Build und Tests",
+                "content": "Nachdem du deine Abgabe eingereicht hast, werden Build und Tests von Artemis automatisch gestartet."
+            },
+            "buildStatus": {
+                "headline": "Buildstatus",
+                "content": "Hier kannst jetzt den Buildstatus einsehen, und erkennen welche Aufgaben die Tests bestanden und erfolgreich gelöst wurden und welche nicht."
```

Comment:
```suggestion
                "content": "Hier kannst jetzt den Buildstatus einsehen, und erkennen welche Tests die Aufgaben bestanden haben und somit erfolgreich gelöst wurden und welche nicht."
```

Reply:
I would also remind them, that there could be tests run after the due date, so that they should always double check their submission and a 100% grade is therefore not always guaranteed.

Reply:
Thanks for the suggestion! I added this hint